src/mongocxx/search_index_view.cpp

```diff
@@ -0,0 +1,105 @@
+    return _get_impl().list(&session, name, options);
+}
+
+bsoncxx::stdx::optional<bsoncxx::string::view_or_value> search_index_view::create_one(
+    bsoncxx::string::view_or_value name, const bsoncxx::document::view_or_value& definition) {
+    return create_one(search_index_model(name, definition));
+}
+
+bsoncxx::stdx::optional<bsoncxx::string::view_or_value> search_index_view::create_one(
+    const client_session& session,
+    bsoncxx::string::view_or_value name,
+    const bsoncxx::document::view_or_value& definition) {
+    return create_one(session, search_index_model(name, definition));
+}
+
+bsoncxx::stdx::optional<bsoncxx::string::view_or_value> search_index_view::create_one(
+    const search_index_model& model) {
+    return _get_impl().create_one(nullptr, model);
+}
+
+bsoncxx::stdx::optional<bsoncxx::string::view_or_value> search_index_view::create_one(
+    const client_session& session, const search_index_model& model) {
+    return _get_impl().create_one(&session, model);
+}
+
+std::vector<bsoncxx::string::view_or_value> search_index_view::create_many(
+    const std::vector<search_index_model>& models) {
+    auto response = _get_impl().create_many(nullptr, models);
+    return _create_many_helper(response["indexesCreated"].get_array().value);
+}
+
+std::vector<bsoncxx::string::view_or_value> search_index_view::create_many(
+    const client_session& session, const std::vector<search_index_model>& models) {
+    auto response = _get_impl().create_many(&session, models);
+    return _create_many_helper(response["indexesCreated"].get_array().value);
+}
+
+std::vector<bsoncxx::string::view_or_value> search_index_view::_create_many_helper(
+    bsoncxx::array::view created_indexes) {
+    std::vector<bsoncxx::string::view_or_value> search_index_names;
+    for (auto&& index : created_indexes) {
+        search_index_names.push_back(
+            bsoncxx::string::to_string(index.get_document().value["name"].get_string().value));
+    }
+    return search_index_names;
+}
+
+void search_index_view::drop_one(bsoncxx::string::view_or_value name) {
+    _get_impl().drop_one(nullptr, name);
+}
+
+void search_index_view::drop_one(const client_session& session,
+                                 bsoncxx::string::view_or_value name) {
+    _get_impl().drop_one(&session, name);
+}
+
+void search_index_view::update_one(bsoncxx::string::view_or_value name,
+                                   const bsoncxx::document::view_or_value& definition) {
+    _get_impl().update_one(nullptr, name, definition);
+}
+
+void search_index_view::update_one(const client_session& session,
+                                   bsoncxx::string::view_or_value name,
+                                   const bsoncxx::document::view_or_value& definition) {
+    _get_impl().update_one(&session, name, definition);
+}
+
+search_index_view::impl& search_index_view::_get_impl() {
+    return *_impl;
+}
+MONGOCXX_INLINE_NAMESPACE_END
+}  // namespace mongocxx
```

Comment:
```suggestion
}  // namespace mongocxx

```
Missing EOF newline.